src/components/navigation/navigation.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  export let kind: 'vertical' | 'horizontal' = 'vertical'
+</script>
+
+<div
+  class="leo-navigation"
+  class:isVertical={kind === 'vertical'}
```

Comment:
optional:

```suggestion
  class:is-vertical={kind === 'vertical'}
```

or 
```suggestion
  class="leo-navigation {kind}"
```

we change between camelCase and kebab-case everywhere so this is 100% optional :laughing: